boards/x86/qemu_x86/qemu_x86_jailhouse_defconfig

```diff
@@ -0,0 +1,21 @@
+CONFIG_X86=y
+CONFIG_SOC_IA32=y
+CONFIG_BOARD_QEMU_X86=y
+CONFIG_CPU_MINUTEIA=y
+CONFIG_IA32_LEGACY_IO_PORTS=y
+CONFIG_JAILHOUSE=y
+CONFIG_JAILHOUSE_X2APIC=y
+CONFIG_PHYS_LOAD_ADDR=0x0
+CONFIG_PHYS_RAM_ADDR=0x0
+CONFIG_IOAPIC_MASK_RTE=n
+CONFIG_PIC_DISABLE=n
+CONFIG_LOAPIC=y
+CONFIG_LOAPIC_TIMER=y
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_NS16550=y
+CONFIG_UART_CONSOLE=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_XIP=n
+CONFIG_X86_MMU=y
+CONFIG_HW_STACK_PROTECTION=n
```

Comment:
any reason why we're disabling stack protection?

Reply:
I have to figure out why `mmu_tables` section wants to end up at an address we have to use here:
`section boot LMA [000000000000fff0,000000000000ffff] overlaps section mmu_tables LMA [000000000000f1f4,0000000000013fff]`. Can this come later? Any clues?

Reply:
i'd really rather this get root-caused. the problem is likely in the x86 linker script.

Reply:
What the patch does WRT boot section is tricky too: it injects that boot section with that hardcoded LMA (that is Jailhouse's entry point, we can't escape that) and the rest of the linker script is not actually well informed of that. Now wondering how to proceed better. Maybe shift LMA of all data sections to be after that?

Reply:
I've been working on this on partial time, but here goes. I've reached people at freenode/##binutils and even they confirm that there is no way/command to tell the linker: "skip this region when assigning LMAs, to account for a hole". We could have 3 slices at ia32/linker.ld (low, boot, high), but again the linker has no support for sections auto-packing, with redirections for more than one memory region via the > operator. We could, finally, have only 2 memory regions there (ram_low, ram) to try and have as few sections as possible patched (only up to the text one in the lower one), but the linker is messing big time with VMAs, still trying to use some that were in the first mem. region in the other. This is very frustrating :/

Reply:
Solved after interaction with Jailhouse upstream. Now for the last issue.